framework/resources/Functional/s3/fpushdown/fpushdown.json

```diff
@@ -0,0 +1,23 @@
+{
+  "testId": "parquet_filter_pushdown_on_s3",
+  "type": "group",
+  "description": "Testing parquet filter pushdown on the S3 storage",
+  "submit-type": "jdbc",
+  "categories": [
+    "functional"
+  ],
+  "dependencies" : ["s3"],
+  "matrices": [
+    {
+      "query-file": ".*.sql.explain",
+      "schema": "s3.tmp",
+      "output-format": "tsv",
+      "expected-file": ".*.plan",
+      "verification-type": [
+        "regex"
+      ]
+    }
+  ],
+  "datasources": []
```

Comment:
Don't these need any data to work on? If it's depending on some other test suite, then please add those copy / gen scripts here as well. We want to make sure each test suite can be executed independently.